corehq/apps/analytics/static/analytics/js/drift.js

```diff
@@ -0,0 +1,56 @@
+/* global _, Array, window */
+
+/**
+ * Instantiates the Drift analytics and customer support messaging platform.
+ */
+hqDefine('analytics/js/drift', function () {
+    'use strict';
+    var _get = hqImport('analytics/js/initial').getFn('drift'),
+        _global = hqImport('analytics/js/initial').getFn('global'),
+        logger = hqImport('analytics/js/logging').getLoggerForApi('Drift'),
+        _utils = hqImport('analytics/js/utils'),
+        _data = {},
+        _drift = {};
+
+    var __init__ = function () {
+        _drift = window.driftt = window.drift = window.driftt || [];
+        if (!_drift.init && !_drift.invoked ) {
+            _drift.methods = [ "identify", "config", "track", "reset", "debug", "show", "ping", "page", "hide", "off", "on" ];
+            _drift.factory = function (methodName) {
+               return function() {
+                    var methodFn = Array.prototype.slice.call(arguments);
+                    methodFn.unshift(methodName);
+                    _drift.push(methodFn);
+                    return _drift;
+                };
+            };
+            _.each(_drift.methods, function (methodName) {
+                _drift[methodName] = _drift.factory(methodName);
```

Comment:
Picky, but I think it'd be clearer to put the guts of `factory` in here rather than pulling them into a separate function.